src/main/java/bio/terra/workspace/app/controller/ControlledGcpResourceApiController.java

```diff
@@ -79,17 +78,14 @@ public ControlledGcpResourceApiController(
             .bucketName(body.getGcsBucket().getName())
             .build();
 
-    // This value is serialized in Stairway, so we avoid making a generic list.
     ControlledResourceIamRoleList privateRoles =
-        (ControlledResourceIamRoleList)
-            Optional.ofNullable(body.getCommon().getPrivateResourceUser())
-                .map(
-                    user ->
-                        user.getPrivateResourceIamRoles().stream()
-                            .map(ControlledResourceIamRole::fromApiModel)
-                            .collect(toCollection(ArrayList::new)))
-                .orElse(null);
-
+        Optional.ofNullable(body.getCommon().getPrivateResourceUser())
+            .map(
+                user ->
+                    user.getPrivateResourceIamRoles().stream()
+                        .map(ControlledResourceIamRole::fromApiModel)
+                        .collect(toCollection(ControlledResourceIamRoleList::new)))
+            .orElse(null);
```

Comment:
generally nullable collections is a bit of an anti-pattern. Is `null` semantically distinct from an empty list here? If not, consider an empty list.